### ISSUE TITLE
Pardot Form Endpoint Not Visible on Reload

### ISSUE THREAD

**Issue by codyjames** [verbb/formie#1123]
I'm so glad this got added just in time for a client project!

One thing I'm noticing though is that when I add an Endpoint URL to a form, it saves correctly (and seems to work), but on reload the Endpoint URL field is empty. So, I'm guessing the value isn't correctly being pulled in to the field.

_Originally posted by @codyjames in https://github.com/verbb/formie/issues/1015#issuecomment-1277990544_

**Comment by codyjames** [comment]
I want to note that this also causes the form to be unable to be saved without disabling the Enable Form Handler or re-adding the Endpoint URL each time you need to edit the form.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by codyjames** [comment]
@engram-design Thanks so much, Josh!

**Comment by engram-design** [comment]
Fixed in [1.6.15](https://github.com/verbb/formie/releases/tag/1.6.15)
